test/integration/generate-renderer.spec.js

```diff
@@ -28,10 +28,10 @@ describe('when a class breaks classification passed in', () => {
       data = _.cloneDeep(ProviderStatsClassBreaks)
     })
     describe('do not exist', () => {
-      it('should throw an error', () => {
+      it('should return empty object', () => {
         const options = {}
         data.statistics = []
-        generateRenderer.bind(this, data, options).should.throw()
+        generateRenderer(data, options).should.deepEqual({})
```

Comment:
This expectation changed after observing response of ArcGIS server under the same circumstances.